perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -28,14 +28,14 @@ Redirect to homepage unless C<id> parameter in query, in which case redirect to
 sub index : Path('') : Args(0) {
     my ( $self, $c ) = @_;
 
-    my $id = $c->get_param('id');
-
-    my $uri =
-        $id
-      ? $c->uri_for( '/report', $id )
-      : $c->uri_for('/');
-
-    $c->res->redirect($uri);
+    if (my $id = $c->get_param('id')) {
```

Comment:
Looks like .com forces `/report` to redirect to `/reports` anyway, for typos, so we've never used it ourselves.
I don't see it being used anywhere, `/around` does it for e.g. front page form, so yeah, could strip it I assume.